test/unit/ldp/http/SparqlUpdateBodyParser.test.ts

```diff
@@ -21,9 +21,9 @@ describe('A SparqlUpdateBodyParser', (): void => {
 
   it('only accepts application/sparql-update content.', async(): Promise<void> => {
     await expect(bodyParser.canHandle(input)).rejects.toThrow(UnsupportedMediaTypeHttpError);
-    input.request.headers = { 'content-type': 'text/plain' };
+    input.metadata.contentType = 'text/plain';
     await expect(bodyParser.canHandle(input)).rejects.toThrow(UnsupportedMediaTypeHttpError);
-    input.request.headers = { 'content-type': 'application/sparql-update' };
+    input.metadata.contentType = 'application/sparql-update';
     await expect(bodyParser.canHandle(input)).resolves.toBeUndefined();
```

Comment:
Plus also:
- `application/sparql-update;charset=utf-8`
- `application/sparql-update ; foo=bar` or similar

Reply:
Those are in the integration tests since the sparql body parser can assume the input it gets has already been through the parser which removes this (until #458 is handled at least).

Reply:
Done